src/mono/netcore/System.Private.CoreLib/src/System/RuntimeType.Mono.cs

```diff
@@ -422,14 +422,14 @@ private static bool FilterApplyPrefixLookup(MemberInfo memberInfo, string? name,
             RuntimeMethodInfo method, BindingFlags bindingFlags, CallingConventions callConv, Type[]? argumentTypes)
         {
             // Optimization: Pre-Calculate the method binding flags to avoid casting.
-            return FilterApplyMethodBase(method, method.BindingFlags, bindingFlags, callConv, argumentTypes);
+            return FilterApplyMethodBase(method, 0, bindingFlags, callConv, argumentTypes);
```

Comment:
If this argument is unused we should just get rid of it. This method is internal to our RuntimeType implementation.